--- ../verrou_repo/valgrind-3.22.0+verrou-dev/verrou/Makefile.am	2024-02-05 13:01:38.251284608 +0100
+++ orig/Makefile.am	2024-02-05 15:41:40.440924957 +0100
@@ -41,6 +41,13 @@
 	interflop_backends/backend_checkdenorm/interflop_checkdenorm.cxx
 
 
+##---------------------------------------------------------------------------##
+##--------------- GENERATED AUTOMAKE FOR ADDITIONAL BACKENDS ----------------##
+
+##------------ END OF GENERATED AUTOMAKE FOR ADDITIONAL BACKENDS ------------##
+##---------------------------------------------------------------------------##
+
+
 
 VERROU_FLAG_CXX = $(subst \
 	-Wmissing-prototypes,,$(subst \
