--- ../verrou_repo/valgrind-3.22.0+verrou-dev/verrou/vr_main.h	2024-02-05 13:01:38.291284609 +0100
+++ orig/vr_main.h	2024-02-05 15:43:39.950925438 +0100
@@ -63,7 +63,13 @@
 
 #define VR_FNNAME_BUFSIZE 4096
 
-typedef enum vr_backend_name{vr_verrou,vr_mcaquad, vr_checkdenorm} vr_backend_name_t;
+typedef enum vr_backend_name{
+  vr_verrou,
+  vr_mcaquad,
+  vr_checkdenorm,
+  /* GENERATED ADDITONAL BACKEND ENUM */
+  /* END OF GENERATED ADDITONAL BACKEND ENUM */
+} vr_backend_name_t;
 
 typedef enum vr_backendpost_name{vr_nopost,vr_checkcancellation, vr_check_float_max} vr_backendpost_name_t;
 
